purchase_sale_inter_company/README.rst

```diff
@@ -0,0 +1,70 @@
+
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================
+Purchase Sale Inter Company
+===========================
+
+This module is usefull if there are multiple companies in the same Odoo database and those companies sell goods or services among themselves.
+It allow to create a sale order in company A from a purchase order in company B.
+
+Imagine you have company A and company B in the same Odoo database.
+Company A purchase goods from company B: company A will create a purchase.order with company B as supplier, and company B will have to create a sale order with company A as customer. This module automate the creation of the sale order in company B.
```

Comment:
please use enumerated list it improves the readibility, such as: 
```
#. Company A purchases goods from company B
#. Company A will create a purchase.order with company B as supplier
#. This module automate the creation of the sale order in company B.
   so that Company B donot have to create a sale order with company A as customer. 
```